src/engine/bank01.asm

```diff
@@ -8142,12 +8142,167 @@ _TossCoin: ; 71ad (1:71ad)
 	ret
 ; 0x72ff
 
-	INCROM $72ff, $7354
+Func_72ff: ; 72ff (1:72ff)
+	ldh [hff96], a
+	ld a, [wDuelType]
+	cp $01
+	ret nz
+	ldh a, [hff96]
+	call SerialSendByte
+	call Func_7344
+	ret
+; 0x7310
+
+Func_7310: ; 7310 (1:7310)
+	ldh [hff96], a
+	ld a, [wDuelType]
+	cp $01
+	jr z, .asm_7338
+.asm_7319
+	call DoFrame
+	call CheckAnyAnimationPlaying
+	jr c, .asm_7319
+	ldh a, [hff96]
+	ret
+	ldh [hff96], a
+	ld a, [wDuelType]
+	cp CONFUSED
```

Comment:
`CONFUSED` is not a valid `wDuelType` value. This is an inappropriate constant to use here.

Reply:
Replaced CONFUSED with DUELTYPE_LINK